README.md

```diff
@@ -211,34 +219,88 @@ Link to other services:
 docker-compose -f docker-compose.yaml -f docker-compose.link.yaml up
 ```
 
+# How to get an output
+
+There are 4 possible outcomes depending on the input:
+
+1. **Input:** A valid payment file.
+**Output:** The transactions are logged, the payment file is moved from `inbound` to `archive`, and each valid payment request is sent to the Topic `PAYMENT_TOPIC_REQUEST`.
+
+2. **Input:** A payment file where the amount of payment requests does not match the batch header.
+**Output:** An error message is logged and the payment file is moved from `inbound` to `quarantine`.
+
+3. **Input:** A payment file with an invalid batch header (e.g. wrong date format)
+**Output:** An error message is logged and the payment file is moved from `inbound` to `quarantine`.
+
+4. **Input:** A payment file where the batch header value matches the payment line values but one or more payment line does not match the corresponding invoice line total.
+**Output:** The valid payment requests are logged and error messages are logged for each invalid request, the payment file is moved from `inbound` to `archive`, each valid payment request is sent to Topic `PAYMENT_TOPIC_ADDRESS`, and an invalid event is sent to `EVENT_TOPIC_ADDRESS` for each invalid payment request.
+
+5. **Input:** A payment file where the batch header sequence number is different from the expected value.
+**Output:** If the sequence number is lower than the expected value the payment file is ignored. If the sequence number is higher than the expected value the payment file is quarantined. A message is logged stating that the payment file has been ignored or quarentined accordingly.
+
+#### Example output message
+
+```
+{
+  "sourceSystem": "SFIP",
+  "frn": 1234567890,
+  "marketingYear": 2022,
+  "paymentRequestNumber": 1,
+  "correlationId":"9e016c50-046b-4597-b79a-ebe4f0bf8505",
+  "invoiceNumber": "SFI12345678",
+  "agreementNumber": "SFI12345",
+  "contractNumber": "SFI12345",
+  "currency": "GBP",
+  "schedule": "Q4",
+  "dueDate": "09/11/2022",
+  "value": 1000.00,
+  "invoiceLines": [{
+    "schemeCode": "80001",
+    "accountCode": "SOS100",
+    "fundCode": "DRD10",
+    "description": "G00 - Gross value of claim",
+    "value": 1000.00
+  }]
+}
+```
+
+# How to stop the Batch Processor
+
+Run the command `docker-compose down`
```

Comment:
The service can be stopped in different ways
- [Bring the service down](#dont-forget-to-change-these)
- [Bring the service down and clear its data](#dont-forget-to-change-these)